Service/Program.cs

```diff
@@ -82,5 +88,5 @@ private static void RunService(ConnectorParams setup)
 public class ConnectorParams
 {
     public bool Service { get; set; }
-    public string WorkDir { get; set; }
+    public string? WorkDir { get; set; }

```

Comment:
the connector still works fine without this defined?

Reply:
correct, it was always an optional field
```
var path = _setup.WorkDir;
if (path == null)
{
    path = Directory.GetParent(AppContext.BaseDirectory)?.Parent?.FullName;
}
```